tests/samples/sample_bundle_stats.json

```diff
@@ -56,7 +56,7 @@
       "id": "index",
       "uniqueId": "2-index",
       "entry": true,
-      "initial": false,
+      "initial": true,
```

Comment:
Does being initial affect the filtering?

Reply:
Yes, its one of the filtering options